gateway/commands.go

```diff
@@ -144,12 +144,12 @@ func (g *Gateway) UpdateStatusCtx(ctx context.Context, data UpdateStatusData) er
 
 // Undocumented
 type GuildSubscribeData struct {
-	Typing     bool              `json:"typing"`
-	Activities bool              `json:"activities"`
-	GuildID    discord.Snowflake `json:"guild_id"`
+	Typing     bool            `json:"typing"`
+	Activities bool            `json:"activities"`
+	GuildID    discord.GuildID `json:"guild_id"`
 
 	// Channels is not documented. It's used to fetch the right members sidebar.
-	Channels map[discord.Snowflake][][2]int `json:"channels"`
+	Channels map[discord.ChannelID][][2]int `json:"channels"`
```

Comment:
I'm pretty sure this is correct but not 100%

Reply:
It is correct.